### PR TITLE
Suggestion page

### PR DESCRIPTION
**Pull request by Elfaram**

### CODE REVIEW

**Review by NtGZeko — DISMISSED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "react-responsive-carousel": "^3.2.23",
     "react-router-dom": "^6.3.0",
     "styled-components": "^5.3.5",
-    "styled-reset": "^4.3.4"
+    "styled-reset": "^4.3.4",
+    "template-fullstack": "file:.."
```
Comment (LuigiMenez) [inline frontend/package.json]: Nope

File: backend/package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
     "express": "^4.17.2",
-    "mysql2": "^2.3.3"
+    "mysql2": "^2.3.3",
+    "template-fullstack": "file:.."
```
Comment (LuigiMenez) [inline backend/package.json]: Nope

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Remove a package template fullstack

**Review by NtGZeko — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by Dash293 — DISMISSED** [review]

**Review by NtGZeko — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Beware: when you want to leave a positive review, don't forget to click the "Approve" button, otherwise it doesn't validate the PR

**Review by NtGZeko — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Beware, you ad a lot of empty lines when solving conflicts. No Evil spotted otherwise !

**Pull request merged**